server/base/src/main/java/org/apache/accumulo/server/tablets/TabletTime.java

```diff
@@ -21,22 +21,20 @@
 
 import org.apache.accumulo.core.client.admin.TimeType;
 import org.apache.accumulo.core.data.Mutation;
+import org.apache.accumulo.core.metadata.schema.MetadataTime;
 import org.apache.accumulo.server.data.ServerMutation;
 import org.apache.accumulo.server.util.time.RelativeTime;
 
 public abstract class TabletTime {
-  public static final char LOGICAL_TIME_ID = 'L';
-  public static final char MILLIS_TIME_ID = 'M';
+  public static final char LOGICAL_TIME_ID = MetadataTime.getCode(TimeType.LOGICAL);
+  public static final char MILLIS_TIME_ID = MetadataTime.getCode(TimeType.MILLIS);
 
   public static char getTimeID(TimeType timeType) {
-    switch (timeType) {
-      case LOGICAL:
-        return LOGICAL_TIME_ID;
-      case MILLIS:
-        return MILLIS_TIME_ID;
-    }
-
-    throw new IllegalArgumentException("Unknown time type " + timeType);
+    /*
+     * switch (timeType) { case LOGICAL: return LOGICAL_TIME_ID; case MILLIS: return MILLIS_TIME_ID;
```

Comment:
As with one of my comments below.  I think this code surrounding the timeType can be removed from this class because it is all provided from the Metadata class now.